CMakeLists.txt

```
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.5)
+project(noether_ros)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(tf2_eigen REQUIRED)
+find_package(noether_tpp REQUIRED)
+find_package(yaml-cpp REQUIRED)
```

I'm a big fan of alphabetizing lists when possible. But that's a style thing.

```suggestion
find_package(ament_cmake REQUIRED)
find_package(geometry_msgs REQUIRED)
find_package(noether_tpp REQUIRED)
find_package(rclcpp REQUIRED)
find_package(tf2_eigen REQUIRED)
find_package(yaml-cpp REQUIRED)
```

---

Done